packages/react/src/Grid/Grid.tsx

```diff
@@ -47,6 +47,8 @@ export type GridProps = {
    * @todo Implement more generally – it will be moved into a theme soon.
    */
   compact?: boolean
+  /** The amount of vertical whitespace between rows of the grid. */
+  gapVertical?: 'small' | 'large'
```

Comment:
Shouldn't you be able to set different values for different breakpoints? Same for the `padding*` props

Reply:
I’m not sure, I hope we can avoid that extra complexity. Basing padding and gap on our responsive horizontal grid gap already helps it behave on narrow and wide screens.

I can imagine we’ll end up with other sizes than the grid gutter. Should we research that first or can we merge this initial approach?

Reply:
I understand. Yes, the spacing approach is the main concern popping up in the [prototypes](https://github.com/Amsterdam/design-system-prototypes) currently. It’s the very next thing we should tackle.